### ISSUE TITLE
Extend getty aat to include facets on hierarchy top concepts

### ISSUE THREAD

**Issue by elrayle** [cul-it/qa_server#27]
### Description:

There is an additional request to facet aat on top hierarchy concepts. Dave has implemented the subset at URL: http://deep-thought.slis.uiowa.edu:8081/ld4l_services/getty_aat.jsp

The QA config needs to be updated to support the faceted hierarchy concepts.

This will be addressed in QA either as a sub-authority or as a simple additional param for aat. More exploration is needed.

### Related Work:

Issue #7 Add Getty vocabularies (PR #22)
Issue #24 Add Getty configs to linked_data_authorities project

**Comment by elrayle** [comment]
Using Subauthorities for hierarchy levels.  The names of the subauthorities are long.  They encode the hierarchy structure and provide the labels that can be used in a select list.

For example,

* `Activities` is a top level
* `Activities__Disciplines` has `Activities` as the top level and `Disciplines` as the second level
* `Activities__Physical_and_Mental` has `Activities` as the top level and `Physical and Mental` as the second level

Parsing...
2 underscores `__` separates a top level from second level
1 underscore `_` indicates a space in the label

```
"Activities"
"Activities__Disciplines"
"Activities__Events"
"Activities__Functions"
"Activities__Physical_and_Mental"
"Activities__Processes_and_Techniques"
"Activities__activities"
"Agents"
"Agents__Living_Organisms"
"Agents__Organizations"
"Agents__People"
"Agents__agents"
"Associated_Concepts"
"Associated_Concepts__Associated_Concepts"
"Brand_Names"
"Brand_Names__Brand_Names"
"Materials"
"Materials__Materials"
"Objects"
"Objects__Built_Environment"
"Objects__Components"
"Objects__Furnishings_and_Equipment"
"Objects__Object_Genres"
"Objects__Object_Groupings and Systems"
"Objects__Visual_and_Verbal_Communication"
"Physical_Attributes"
"Physical_Attributes__Attributes_and_Properties"
"Physical_Attributes__Color"
"Physical_Attributes__Conditions_and_Effects"
"Physical_Attributes__Design_Elements"
"Styles_and_Periods"
"Styles_and_Periods__Styles_and_Periods"
```

**Comment by elrayle** [comment]
The following queries test the use of hierarchies using subauthorities...
* All levels -- http://lookup.ld4l.org/qa/search/linked_data/getty_aat_ld4l_cache?q=domestic&maxRecords=20
* Top level Agents -- http://lookup.ld4l.org/qa/search/linked_data/getty_aat_ld4l_cache/Agents?q=domestic&maxRecords=20
* Second level People under Agents -- http://lookup.ld4l.org/qa/search/linked_data/getty_aat_ld4l_cache/Agents__People?q=domestic&maxRecords=20

**Comment by elrayle** [comment]
Fixed by PR #29
